Localization/fr-FR_Mods.SpelunkerFilter.hjson

```diff
@@ -1,94 +1,94 @@
+			Label: Liste blanche Personnalisée
+			Tooltip: Les blocs dans cette liste SERONT être en surbrillance. Priorité au-dessus de "{$Configs.SFConfig.Headers.PresetFilter}" et "{$Configs.SFConfig.CustomBlacklist.Label}"
 		}
 
 		CustomBlacklist: {
-			// Label: Custom Blacklist
-			// Tooltip: Tiles in this list WILL NOT be highlighted. Takes precedence over "{$Configs.SFConfig.Headers.PresetFilter}"
+			Label: Liste noire Personnalisée
+			Tooltip: Les blocs dans cette liste ne SERONT pas en surbrillance. Priorité au-dessus de "{$Configs.SFConfig.Headers.PresetFilter}"
 		}
 
 		ToggleGems: {
-			// Label: Toggle Gems
-			// Tooltip: Toggles all gems
+			Label: Activer/Désactiver gemmes
+			Tooltip: Active/Désactive toutes les gemmes
 		}
 
 		TogglePreHMMetalOres: {
-			// Label: Toggle Pre-HM Ores
-			// Tooltip: Toggles all pre-hardmode metal ores
+			Label: Activer/Désactiver minerais Pré-Hardmode
+			Tooltip: Active/Désactive tous les minerais du Pré-Hardmode
 		}
 
 		ToggleHMMetalOres: {
-			// Label: Toggle HM Ores
-			// Tooltip: Toggles all hardmode metal ores
+			Label: Activer/Désactiver minerais Hardmode
+			Tooltip: Active/Désactive tous les minerais du Hardmode
 		}
 
-		// Pots.Label: Pots
-		// DesertFossil.Label: Desert Fossil
-		// FossilOre.Label: Sturdy Fossil
-		// Copper.Label: Copper Ore
-		// Tin.Label: Tin Ore
-		// Iron.Label: Iron Ore
-		// Lead.Label: Lead Ore
-		// Silver.Label: Silver Ore
-		// Tungsten.Label: Tungsten Ore
-		// Gold.Label: Gold Ore
-		// Platinum.Label: Platinum Ore
-		// Meteorite.Label: Meteorite Ore
-		// Containers.Label: Containers
-		// FakeContainers.Label: Fake Containers
-		// Containers2.Label: Containers 2
-		// FakeContainers2.Label: Fake Containers 2
-		// Heart.Label: Life Crystal
-		// ManaCrystal.Label: Mana Crystal
-		// Cobalt.Label: Cobalt Ore
-		// Palladium.Label: Palladium Ore
-		// Mythril.Label: Mythril Ore
-		// Orichalcum.Label: Orichalcum Ore
-		// Adamantite.Label: Adamantite Ore
-		// Titanium.Label: Titanium Ore
-		// Chlorophyte.Label: Chlorophyte Ore
-		// DyePlants.Label: Dye Plants
-		// LifeFruit.Label: Life Fruit
-		// Sapphire.Label: Sapphire
-		// Ruby.Label: Ruby
-		// Emerald.Label: Emerald
-		// Topaz.Label: Topaz
-		// Amethyst.Label: Amethyst
-		// Diamond.Label: Diamond
-		// MatureHerbs.Label: Mature Herbs
-		// BloomingHerbs.Label: Blooming Herbs
-		// Statues.Label: Statues
-		// ExposedGems.Label: Exposed Gems
-		// Painting3X3.Label: Paintings (3x3)
-		// Painting6X4.Label: Paintings (6x4)
-		// Painting2X3.Label: Paintings (2x3)
-		// Painting3X2.Label: Paintings (3x2)
-		// AlphabetStatues.Label: Alphabet Statues
-		// MushroomStatue.Label: Mushroom Statue
-		// CatBast.Label: Bast Statue
-		// BoulderStatue.Label: Boulder Statue
-		// AmberStoneBlock.Label: Amber
+		Pots.Label: Jarres
```

Comment:
This and the rest of the labels: Make sure it's properly translating plural when it's explicitely plural in english, and otherwise singular. For example Pots and Plaintings is plural, but Ruby and Iron Ore isn't

Reply:
For that, I fixed Painting that is not in plurial in the internalname to "Peinture".